website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,193 @@
+3. Once you've auto-formatted, you'll see a message confirming the outcome. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/prettier.gif" width="95%" title="Format YAML, Markdown, and JSON files using Prettier."/>
+
+### Format Python
+
+To format your Python code, dbt Cloud integrates with [Black](https://black.readthedocs.io/en/latest/), which is an uncompromising Python code formatter.
+
+1. To enable formatting, make sure you open a `.py` file.
+2. In the console section (located below the **File editor**), select the **Format** button to auto-format your code in the **File editor**. 
+3. Once you've auto-formatted, you'll see a message confirming the outcome. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/python-black.gif" width="95%" title="Format Python files using Black."/>
+
+## Troubleshooting
+
+<details>
+<summary>Can I nest <code>.sqlfluff</code> files?</summary>
+
+To ensure optimal code quality, consistent code, and styles, it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. Having multiple files can result in various different SQL styles in your project. <br /><br />
+
+However, you can customize and include an additional child `.sqlfluff` configuration file within specific subfolders of your dbt project. <br /><br />By nesting a `.sqlfluff` file in a subfolder, SQLFluff will apply the rules defined in that subfolder's configuration file to any files located within it. The rules specified in the parent `.sqlfluff` file will be used for all other files and folders outside of the subfolder. This hierarchical approach allows for tailored linting rules while maintaining consistency throughout your project. Refer to [SQLFluff documentation](https://docs.sqlfluff.com/en/stable/configuration.html#configuration-files) for more info.
+
+</details>
+<details>
+<summary>Can I nest <code>.sqlfluff</code> files?</summary>
+
+To ensure optimal code quality, it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. However, you have the flexibility to customize and include an additional child `.sqlfluff` configuration file within specific subfolders of your dbt project. <br /><br />By nesting a `.sqlfluff` file in a subfolder, SQLFluff will apply the rules defined in that subfolder's configuration file to any files located within it. For all other files and folders outside of the subfolder, the rules specified in the parent `.sqlfluff` file will be used. This hierarchical approach allows for tailored linting rules while maintaining consistency throughout your project. Refer to [SQLFluff documentation](https://docs.sqlfluff.com/en/stable/configuration.html#configuration-files) for more info.
+
+</details>
+
+
+## Next steps
+
+- [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)
+- [Tips and tricks](/docs/cloud/dbt-cloud-ide/dbt-cloud-tips)
```

Comment:
looking at the deploy preview: 

![Screenshot 2023-06-08 at 8 27 33 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/10b08af5-baa3-4723-8c94-a936e2e54bed)

- is there a duplicate in the Troubleshooting section? sugg removing the dupe 
- the links in the Next steps section look to be the same as the links in the footer. sugg removing the Next steps section

Reply:
done, thank you Ly!